src/media/Router.ts

```diff
@@ -143,6 +147,66 @@ export class Router extends EventEmitter {
 		return canConsume;
 	}
 
+	@skipIfClosed
+	public async createActiveSpeakerObserver({
+		interval,
+		appData = {}
+	}: ActiveSpeakerObserverOptions): Promise<ActiveSpeakerObserver> {
+		logger.debug('createActiveSpeakerObserver()');
+
+		const {
+			id,
+		} = await this.connection.request({
+			method: 'createActiveSpeakerObserver',
+			data: {
+				routerId: this.id,
+				interval,
+			}
+		}) as ActiveSpeakerObserverOptions;
+
+		const activeSpeakerObserver = new ActiveSpeakerObserver({
+			router: this,
+			connection: this.connection,
+			id,
+			appData,
+		});
+
+		this.activeSpeakerObservers.set(id, activeSpeakerObserver);
+		activeSpeakerObserver.once('close', () => this.activeSpeakerObservers.delete(id));
+
+		return activeSpeakerObserver;
+	}
+
+	@skipIfClosed
+	public async createAudioLevelObserver({
```

Comment:
I think we need to call this method (in constructor?) or middleware will never handle the messages from media-node, since there is no `AudioLevelObserver` on the router.